site/gatsby-site/i18n/locales/es/translation.json

```diff
@@ -75,6 +74,19 @@
   "Flagged Incidents": "Reportes Denunciados",
   "Word Counts": "Contadores de palabras",
   "Read More": "Leer Mas",
+  "Email address": "Correo electrónico",
+  "or": "o",
+  "An unknown error has ocurred": "Ha ocurrido un error desconocido",
+  "Invalid email": "Correo electrónico inválido",
+  "Passwords must match": "Las contraseñas deben coincidir",
+  "Required": "Requerido",
+  "Password must be at least 6 characters long": "La contraseña debe tener al menos 6 caracteres",
+  "Password must be less than 128 characters long": "La contraseña debe tener menos de 128 caracteres",
+  "user not found": "usuario no encontrado",
```

Comment:
Lowercase looks weird 🤔 

Any reason to not use capital case?

Reply:
Yes, that looks weird. The reason is that Realm returns the error messages without capital case and I want to translate them to display in the error toast message.

Reply:
I added `tw-capitalize` tailwind class on each toast error message to avoid this.